apps/examples/src/examples/speech-bubble/SpeechBubble/SpeechBubbleUtil.tsx

```diff
@@ -18,13 +18,15 @@ import {
 	resizeBox,
 	structuredClone,
 } from '@tldraw/tldraw'
-import { STROKE_SIZES } from '@tldraw/tldraw/src/lib/shapes/shared/default-shape-constants'
```

Comment:
We don't want to do this type of deep importing. It would not work outside of our monorepo, plus (IMO) we'd rather have folks copy this stuff out.